test/IntegrationTests/Helpers/TestHttpServer.NetFramework.cs

```diff
@@ -64,6 +64,7 @@ public TestHttpServer(ITestOutputHelper output, Action<HttpListenerContext> requ
             {
                 _listener.Close(); // a new listener is created in the beginnning of the loop
                 retries--;
+                continue;
```

Comment:
Actually, continue needs to be conditional on the number of retries...

Reply:
This is funny as there is `when (retries > 0)` in the `catch` :wink: